src/Core/Geometry/Cylinder.hpp

```diff
@@ -6,7 +6,7 @@
 
 #include <memory>
 
-
+/** class to generate cylinders */
```

Comment:
Please try to be consistent in the style of comments, for example using the longer blocks to document classes.

```suggestion
/**
 * Class to represent cylinders.
 */
```